packages/runtime/src/templates/getApiHandler.ts

```diff
@@ -66,8 +66,6 @@ const makeHandler = (conf: NextConfig, app, pageRoot, page) => {
 
     const NextServer: NextServerType = getNextServer()
     const nextServer = new NextServer({
-      // We know we're just an API route, so can enable minimal mode
```

Comment:
That's a lie. It's used by any site deployed to Vercel. It disables various features, such as on-disk caching that don't work in lambdas. It also disables lots of things that Vercel does at the edge instead. I turned it on so it didn't need to do routing (which we didn't need because it was one route per function). However the thing I missed is that it also disabled custom headers.